DESCRIPTION

```diff
@@ -45,3 +45,4 @@ Collate:
     'shortcuts.R'
     'wildcards.R'
     'zzz.R'
+Config/Needs/website: r-lib/pkgdown
```

Comment:
Needed because CRAN version has NEWS.md parsing bug. Should be released next week.